From d7018f2b143bd0456bb57471c0408d24b94a97d3 Mon Sep 17 00:00:00 2001
From: GearKite <onyx-commute-quit@duck.com>
Date: Fri, 20 Mar 2026 21:32:30 +0200
Subject: [PATCH] return on receiveDeleteForEveryone

---
 ts/util/deleteForEveryone.preload.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ts/util/deleteForEveryone.preload.ts b/ts/util/deleteForEveryone.preload.ts
index b9aafb675..719c20e7a 100644
--- a/ts/util/deleteForEveryone.preload.ts
+++ b/ts/util/deleteForEveryone.preload.ts
@@ -31,6 +31,7 @@ export async function receiveDeleteForEveryone(
   >,
   { shouldPersist = true }: { shouldPersist?: boolean } = {}
 ): Promise<void> {
+  return;
   const conversation = window.ConversationController.get(
     message.get('conversationId')
   );
--
2.51.2


